src/index.js

```diff
@@ -16,18 +16,8 @@ governing permissions and limitations under the License.
 
 /* global Observable */ // for linter
 
-const {
-  reduceError,
-  appendQueryParams,
-  parseLinkHeader,
-  parseRetryAfterHeader,
-  genErrorResponse,
-  getProperPayload
-} = require('./helpers')
-const {
-  verifyDigitalSignature,
-  isTargetRecipient
-} = require('./signatureUtils')
+const helpers = require('./helpers')
```

Comment:
Is this possible? This would reduce a lot of verbosity in the code as well?
```java
const helpers = require('./helpers').exportFunctions;
```

Reply:
will check now.

Reply:
this works. will make the change